src/components/NavBar.vue

```diff
@@ -1,31 +1,113 @@
 <template>
-	<nav>
+	<div id="nav-bar">
 		<img src="../assets/logo.svg">
-		<a href="#about">About</a>
-		<a href="#speakers">Speakers</a>
-		<a href="#testimonials">Testimonials</a>
-		<a href="#sponsors">Sponsors</a>
-		<a href="#faq">FAQ</a>
-		<!-- <router-link to='/'>Home</router-link> -->
-		<!-- <router-link :to='{name: "Blog", params: {post: "main"}}'>Read our blog</router-link> -->
-	</nav>
+
+		<label v-if="isMobile" @click="openMenu">&#9776;</label>
```

Comment:
This label doesn't appear correctly on mobile:
![image](https://user-images.githubusercontent.com/5107795/110257503-3b1ec580-7f6c-11eb-9400-51ceca6c6588.png)
Figma has it right-aligned with some padding away from the screen. Could this be done?

Reply:
Also uh do we still not have Montserrat in the website? It's still not loading for me, as you can see in the screenshot.

Reply:
Corrected that. We don't seem to have a monterserrat link in the index.html. I'll add that in another push